src/reporting/market.se

```diff
@@ -211,7 +215,6 @@ def tryFinalizeAutomatedReport():
         return(0)
     self.finalPayoutDistributionHash = self.automatedReporterPayoutDistributionHash
     self.finalizationTime = block.timestamp
-    self.privateRewardBondHolders()
     self.reportingWindow.updateMarketPhase()
```

Comment:
Oh, sorry.  This could should probably be extracted into a helper function that just looks at the final payout distribution, compares it to each of the dispute bonds, and sends REP to each dispute bonds and if they have a matching payout distribution then send their REP to the winning reporting token.

No matter how the event finalized, _all_ bonds should be evaluated against the _final_ outcome.